charts/icm/values-test-local.yaml

```diff
@@ -0,0 +1,53 @@
+icm-as:
+  nodeSelector: []
+  license:
+    secret:
+      name: intershop-license
+  persistence:
+    sites:
+      size: 1Gi
+      type: local
+      local:
+        path: /run/desktop/mnt/host/d/tmp/pv/sites
+    customdata:
+      enabled: true
+      existingClaim: icm-11-test-local-testdata-pvc
+      mountPoint: /data
+  datadog:
+    enabled: false
+  mssql:
+    enabled: true
+    acceptEula: "Y"
+    persistence:
+      data:
+        size: 1Gi
+        type: local
+        local:
+          path: /run/desktop/mnt/host/d/tmp/pv/mssql/data
+      backup:
+        size: 1Gi
+        type: local
+        local:
+          path: /run/desktop/mnt/host/d/tmp/pv/mssql/backup
+
+icm-web:
+  nodeSelector: []
+  persistence:
+    pagecache:
+      size: 1Gi
+      type: local
+      local:
+        path: /run/desktop/mnt/host/d/tmp/pv/pagecache
+    customdata:
+      enabled: false
+
+testrunner:
+  nodeSelector: []
+  persistence:
+    testdata:
+      size: 1Gi
+      type: local
+      local:
+        dir: /run/desktop/mnt/host/d/tmp/pv/testdata
+  datadog:
+    enabled: false
```

Comment:
why is datadog.enabled==false ? values-iste_linux.tmpl defines datadog.enabled=true

Reply:
* `values-iste_linux.tmpl` is the one ISTE uses as a template file where envs would be replaced before testsuite starts
* `values-test-local.yaml` is for the local test execution, it will be used automatically if `start-test-local.sh` is called

Reply:
Okay, additionally to say.. `values-iste_linux.tmpl` and `values-test-local.yaml` are both used for the local execution but `values-test-local.yaml` is stronger